src/utils/themes.js

```diff
@@ -95,6 +95,10 @@ export const semanticLightThemeColors = {
   hazardIconColor: '#8b939e',
   refreshControl: '#0000004C',
   interjectionPointText: '#0a1427',
+  walletConnectMessage: '#818eb3',
+  swiperButtonPrimaryBackground: lightThemeColors.basic000,
+  swiperButtonThumbBackground: baseColors.white,
+  handleBarBackground: '#ededed',
```

Comment:
This is more a color of handle bar than it's background color. You apply it using `background-color` but that an implementation details that might change.

```suggestion
  handleBarColor: '#ededed',
```